packages/template-retail-react-app/app/pages/checkout/index.test.js

```diff
@@ -62,48 +63,9 @@ const WrappedCheckout = () => {
 // Set up and clean up
 beforeEach(() => {
     global.server.use(
-        rest.get('*/customers/:customerId/baskets', (req, res, ctx) => {
-            return res(
-                ctx.json({
-                    baskets: [keysToCamel(ocapiBasketWithItem)]
-                })
-            )
-        }),
-
-        // mock product variant detail
-        rest.get('*/products', (req, res, ctx) => {
-            return res(ctx.json(productsResponse))
-        }),
-
-        // mock available shipping methods
-        rest.get('*/shipments/me/shipping_methods', (req, res, ctx) => {
-            return res(ctx.json(mockShippingMethods))
-        }),
-
-        // mock available payment methods
-        rest.get('*/baskets/:basketId/payment_methods', (req, res, ctx) => {
-            return res(ctx.json(mockPaymentMethods))
-        }),
-
         // mock product details
         rest.get('*/products', (req, res, ctx) => {
             return res(ctx.json({data: [{id: '701642811398M'}]}))
```

Comment:
Within `beforeEach`, it turns out that this time we need only this 1 mocked response.